TrelloAPI/BaseRestClient.cs

```diff
@@ -0,0 +1,17 @@
+﻿using RestSharp;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TrelloAPI
+{
+    public static class BaseRestClient
+    {
+        private static RestClient _restClient;
+        public static RestClient RestClientInstance { get => _restClient; }
```

Comment:
Why not just 
`public static RestClient RestClientInstance { get; private set; }`
? without the  `_restClient` field